game_view.h

```diff
@@ -37,9 +38,25 @@ class game_view
   ///Gets a const ref to m_game
   const game& get_game() const noexcept {return m_game; }
 
+  /// Get next view
+  view_mode get_next_view() const
+  {
+    return m_next_view;
+  }
+
+  // Get the size of the window
+  const sf::Vector2f get_window_size() { return m_window_size; }
+
+  // Get the window's state, for testing purposes only
+  bool is_window_open() { return m_window.isOpen(); }
+
   ///Gets a ref to m_game
   game& get_game() noexcept {return m_game; }
 
+#ifdef VIEW_SWITCH
+  // Returns what view should come next
+  const view_mode& what_next() noexcept { return m_next_view; }
```

Comment:
Hey another one :) Add another const here to make the world a better place!

Reply:
```suggestion
  const view_mode& what_next() const noexcept { return m_next_view; }
```